railgun/apis/configuration/v1beta1/ingress_rules.go

```diff
@@ -0,0 +1,44 @@
+package v1beta1
+
+// IngressRulePortOnly represents a rule to apply against incoming requests
+// wherein no Host matching is available for request routing, only the port
+// is used to match requests.
+type IngressRulePortOnly struct {
```

Comment:
naming suggestion: call this type `UDPIngressRule`.

Why?
- This type is likely to be extended in the future
- This type is going to configure UDP ingresses and nothing more (or am I wrong?)
- The "port only" claim may age badly

Reply:
:+1: 43c8beea0136f0de992d2184a8c2e23a50df9651